tests/phpunit/tests/test-accept-languages-collection.php

```diff
@@ -44,55 +64,55 @@ public function test_parse_all_language_tags() {
 
 		$accept_languages = PLL_Accept_Languages_Collection::from_accept_language_header( $http_header );
 
-		$this->assertEquals( 3, count( $this->get_accept_languages_array( $accept_languages ) ) );
+		$this->assertSame( 3, count( $this->get_accept_languages_array( $accept_languages ) ) );
```

Comment:
Since you want to improve this...
```suggestion
		$this->assertCount( 3, $this->get_accept_languages_array( $accept_languages ) );
```